### ISSUE TITLE
Upgrade SvelteKit to v2

### ISSUE THREAD

**Issue by maiertech** [impierce/identity-wallet#194]
### Description

Upgrade the frontend to Sveltekit v2.

### Motivation

SvelteKit v2 was released in December 2023. The SvelteKit docs do not maintain docs for legacy versions, i.e. when we refer to the SvelteKit docs, there may be discrepancies between how things work in v1 and v2. Upgrading to v2 is also a prerequisite to upgrading Svelte to v5 later in 2024.

### Resources

https://kit.svelte.dev/docs/migrating-to-sveltekit-2

### To-do List

Since we use SvelteKit with adapter-static and none of the features with breaking changes, the upgrade should involve bumping up version numbers only and adjusting configuration files. We can use the official migration script.

**Comment by maiertech** [comment]
Maybe Vite 5 will fix #188?

**Comment by maiertech** [comment]
After upgrade to v2 SvelteKit throws this warning:

```
You have specified a baseUrl and/or paths in your tsconfig.json which interferes with SvelteKit's auto-generated tsconfig.json. Remove it to avoid problems with intellisense. For path aliases, use `kit.alias` instead: https://kit.svelte.dev/docs/configuration#alias
```

Need to fix our `tsconfig.json` to make sure it does not interfere with the one created by SvelteKit.
